src/cli_shared/cli/config.rs

```diff
@@ -1,18 +1,125 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use core::time::Duration;
-use std::{path::PathBuf, sync::Arc};
-
 use crate::chain_sync::SyncConfig;
 use crate::db::db_engine::DbConfig;
 use crate::libp2p::Libp2pConfig;
 use crate::networks::ChainConfig;
-use log::LevelFilter;
-use serde::{Deserialize, Serialize};
+use core::time::Duration;
+use serde::{
+    de::{DeserializeSeed, EnumAccess, Error, Unexpected, VariantAccess, Visitor},
+    Deserialize, Deserializer, Serialize, Serializer,
+};
+use std::{fmt, path::PathBuf, str::FromStr, sync::Arc};
+use tracing::log::LevelFilter;
 
 use super::client::Client;
 
```

Comment:
This doesn't look right. Why do we need to serialize log levels? Maybe removing that should be a separate PR.

For now, let's just
```
#[derive(serde::Serialize, serde::Deserialize)]
#[serde(rename = ...)]
enum LogLevel {
    Off,
    ...
}
```